plugins/blocks/developedBy/templates/block.tpl

```diff
@@ -8,6 +8,10 @@
  * Common site sidebar menu -- "Developed By" block.
  *}
 <div class="pkp_block block_developed_by">
+	<h2 class="title">
```

Comment:
I think this should also have the `.pkp_screen_reader` class since it normally doesn't include a title.

Reply:
Done!